src/puppetlabs/trapperkeeper/services/webserver/jetty9_service.clj

```diff
@@ -1,13 +1,20 @@
 (ns puppetlabs.trapperkeeper.services.webserver.jetty9-service
-  (:import (org.eclipse.jetty.jmx MBeanContainer))
+  (:import (org.eclipse.jetty.jmx MBeanContainer)
+           (java.io File))
```

Comment:
Don't think File is used in this namespace

Reply:
Ack, forgot to remove those when I switched over to cherry-picking these commits.